Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,8 +91,19 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
-            return array;
+            if (array == null || index < 0 || index >= array.Length)
+            {
+                return array;
+            }
+            else
```

Comment:
If you return, else is not needed because the function exits.

Reply:
Modification done. Thanks!